pkg/ingester/ingester.go

```diff
@@ -107,6 +107,18 @@ const (
 
 	// Prefix used in Prometheus registry for ephemeral storage.
 	ephemeralPrometheusMetricsPrefix = "ephemeral_"
+
+	// Label name used to select queried storage type.
+	StorageLabelName            = "__storage__"
```

Comment:
Shall we use something less generic, eg. `__mimir_storage__` here?

Reply:
separate topic: should we prevent the successful ingestion of this label?

Reply:
> should we prevent the successful ingestion of this label?

I think the risk is tiny, and it would complicate our code. I'd leave it as-is.